File: src/InventoryTrack/src/Services/InventoryService/InventoryService.Application/DTOs/Request/Warehouse/CreateWarehouseDto.cs
using InventoryService.Domain.Enums;

namespace InventoryService.Application.DTOs.Request.Warehouse;

public class CreateWarehouseDto
{
    public string Name { get; set; }
    public WarehouseType Type { get; set; }
    public string Location { get; set; }
    public Guid CompanyId { get; set; }
    public Guid ResponsiblePersonId { get; set; }
}

File: src/Client/inventory-track/src/app/models/dto/warehouse/create-warehouse-dto.ts
import { WarehouseType } from './enums/warehouse-type.enum';

export class CreateWarehouseDto {
  name!: string;
  type!: WarehouseType;
  location!: string;
  companyId!: string;
  responsiblePersonId!: null | string;
  accountantId: string | null = null;
}
